### PR TITLE
add css

### PR DESCRIPTION
**Pull request by madarduale**
soo eeg

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*{
+    background-color: aliceblue;
```
Comment (samiirtello) [inline style.css:2]: make color hex

Comment (madarduale) [inline style.css:2]: make hexa

**Review by madarduale — COMMENTED** [review]
mkea heax[lajaknjnak

**Review by samiirtello — COMMENTED** [review]
halkaa say wax ka dhiman yihhin

File: style.css (hunk 1 above)
Comment (madarduale) [reply style.css:2]: waa sxa

Comment (samiirtello) [reply style.css:2]: haaa

Comment (samiirtello) [reply style.css:2]: haye

Comment (madarduale) [reply style.css:2]: amaxad ku darat

Comment (samiirtello) [reply style.css:2]: waxba

**Pull request merged**